e2e/src/app.e2e-spec.ts

```diff
@@ -8,7 +8,7 @@ describe('workspace-project App', () => {
     page = new AppPage();
   });
 
-  it('should display welcome message', async () => {
+  xit('should display welcome message', async () => {
```

Comment:
@carryall What if we remove this out as we don't have this message in screen anymore? 😅

Reply:
✅  Fixed in [0510b27](https://github.com/carryall/survey-challenge/pull/2/commits/0510b2712efb07a0111dfe88925c4299876dbd45)